### PR TITLE
fix(register): adjust and improve broken styling for auth pages

### PR DESCRIPTION
**Pull request by nathan-power**
**What this PR does** 📖
- Fixes styling for register related pages.
- Creates InfoModal.
- Moves the pin-info section in the signup unlock page into the InfoModal. 

**Which issue(s) this PR fixes** 🔨
AP-2022

**Special notes for reviewers** 🗒️

**Additional comments** 🎤

### CODE REVIEW

**Comment by WanderingHogan** [comment]
Lets try to get Nicholas' wallet PR in first if it's ready to go before merging this

**Comment by WanderingHogan** [comment]
actually, there are no conflicting files with his PR, sorry, removing merge blocked tag

File: components/ui/InfoButton/InfoButton.vue [hunk 1]
```diff
@@ -0,0 +1,136 @@
+<template>
```
Comment (aewing) [inline components/ui/InfoButton/InfoButton.vue:1]: Nice. Very clean.

**Review by molimauro — CHANGES_REQUESTED** [review]
Look good i just have a minor question :) Why have we added a bounce effect on the info modal?

**Comment by nathan-power** [comment]
> Looks good i just have a minor question :) Why have we added a bounce effect on the info modal?

@molimauro 
Good call. I've replaced the internal animation styling of InfoModal by wrapping the UiModal component inside of it. We can keep the animation styling centralized this way as well.

**Comment by nathan-power** [comment]
> Looks good i just have a minor question :) Why have we added a bounce effect on the info modal?

@molimauro
Good call. I've replaced the internal animation styling of InfoModal by wrapping the UiModal component inside of it. We can keep the animation styling centralized this way as well.

**Review by molimauro — APPROVED** [review]
Dope, thanks!

**Pull request merged**